src/python/gennylib/genny_auto_tasks.py

```diff
@@ -279,12 +278,15 @@ def construct_all_tasks_json():
         task_name = to_snake_case(base_parts[0])
 
         prepare_environment_vars = get_prepare_environment_vars(task_name, fname)
+        timeout_params = {"exec_timeout_secs": 86400, # 24 hours
+                          "timeout_secs": 86400}
```

Comment:
This is the timeout for no output, often incorrectly referred to as the "test timeout". We should probably lower it to 2 hours or something. If test runs for more than 2 hours, it needs to provide progress updates.

Reply:
2 hours is actually the default in evergreen; I had to set it higher to run InsertRemove since it doesn't output anything. If we want to require progress updates though I'm cool with that.